exhibitions/app/model/exhibition-promo.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import {model} from 'common';
+const {UiComponent, Picture} = model;
+
+export type ExhibitionPromo = UiComponent & {
+  id?: ?String;
+  url?: ?string;
+  title: string;
+  image?: ?Picture;
+  description?: string;
+  start?: ?Date;
+  end?: ?Date;
```

Comment:
The double-nullable is really strange (I assume it's a hack for nullable parameters with default values) - in the case, because this is done dynamically, which it mostly should be done, we can have `{ key: ?value }` - this seems to give us a much clearer account of what our objects look like.

[They offer a solution to this](https://flow.org/en/docs/types/primitives/#toc-function-parameters-with-defaults), which we have sort of used elsewhere, but the use case was hand-cranking data (Wordpress 👊!)  which is using methods with default values.

Could we also make this a [`{||}` exact object type](https://flow.org/en/docs/types/objects/#toc-exact-object-types)?

The UiComponent to was another attempt at something I'm not to sure was successful - do we need it here? If not, let's not use it and maybe resign UiComponent to the depths of the Great Type Cleanup that's due.